docker/graphite-api.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec /sbin/setuser root gunicorn -b 0.0.0.0:8000 -w 2 --timeout 120 --log-level debug graphite_api.app:app >>/var/log/graphite-api.log 2>&1
+exec /sbin/setuser root gunicorn -b 0.0.0.0:8000 -w 2 --timeout 300 --log-level debug graphite_api.app:app >>/var/log/graphite-api.log 2>&1
```

Comment:
Might be better to make the timeout an environment variable with a default so that it can be overridden.